app.py

```diff
@@ -43,6 +43,12 @@ def create_app() -> FlaskApp:
     health = Healthcheck(flask_app)
     health.add_check(FlaskRunningChecker())
     health.add_check(DbChecker(db))
+
+    @flask_app.errorhandler(404)
+    def not_found(error):
+        flask_app.logger.warning("requested URL was not found on the server")
+        return "Requested URL was not found on the server", 404
```

Comment:
`fund-store` is an API - it should return JSON consistently, not just some text, no? Just trying to think about keeping the API output clear.

If we're going to be standardising this in fsd-utils shortly that's great, but might still be good to update this for the interim.

```suggestion
        return jsonify({"code": 404, "error": "Requested URL was not found on the server"}), 404
```

Reply:
Done updated